### ISSUE TITLE
allow comments in config files?

### ISSUE THREAD

**Issue by arkarkark** [tsowell/kobo-btpt#15]
It'd be nice if the config line reader ignored config file lines that start with `#` this would allow me to add comments about the device and which buttons each line is for.

I'm happy to put together a pull request if you like this idea.

**Comment by tsowell** [comment]
Good idea! It would be great if you could put together a pull request.

**Comment by jmcphers** [comment]
Also would be nice if the config file reader ignored blank lines, I had one at the end of my file and couldn't figure out why it wasn't being picked up until I pulled the syslog.
